apps/site/next.constants.mjs

```diff
@@ -143,6 +142,15 @@ export const DEFAULT_ORAMA_QUERY_PARAMS = {
   },
 };
 
+/**
+ * The initial Orama Cloud chat suggestions visible in the empty state of the search box.
+ */
+export const DEFAULT_ORAMA_SUGGESTIONS = [
+  'How to install Node.js?',
+  'How to create an HTTP server?',
+  'Upgrading Node.js version',
```

Comment:
this yields no results

![image](https://github.com/user-attachments/assets/f8ad7cd3-a4a8-4147-9732-12788daf4196)

```suggestion
  'What are the supported versions of Node.js?',
```